src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--mon1"></div>
```

Comment:
you dont need this enywhere
```suggestion
      <div class="calendar__day"></div>
```